alfalfa_worker/jobs/openstudio/lib/variables.py

```diff
@@ -0,0 +1,140 @@
+import json
+from os import PathLike
+from pathlib import Path
+from typing import Dict, List
+from xml.etree import ElementTree
+
+from alfalfa_worker.lib.enums import PointType
+from alfalfa_worker.lib.models import Point, Run
+
+
+class Variables:
+    inputs: Dict[str, Dict]
+    outputs: Dict[str, Dict]
+    day_index: int = 0
+    hour_index: int = 1
+    minute_index: int = 2
+    month_index: int = 3
+
+    def __init__(self, run: Run) -> None:
+        self.run = run
+        self.inputs = self._load_json('**/input_points.json')
+        self.outputs = self._load_json('**/output_points.json')
+
+    def _load_json(self, glob: str) -> List:
+        result = {}
+        files = self.run.glob(glob)
+        for file in files:
+            with open(str(file), 'r') as fp:
+                result.update(json.load(fp))
+
+        return result
+
+    def load_reports(self):
+        self.inputs = self._load_json('**/*_report_inputs.json')
+        self.outputs = self._load_json('**/*_report_outputs.json')
+
+    def generate_variables_cfg(self) -> str:
+        bcvtb_variables_element = ElementTree.fromstring(
```

Comment:
crazy but looks good. If we are going to handle XML a lot, then we should wrap the generation of these into an XML Class of some sort so we can validate.